relay-server/src/envelope.rs

```diff
@@ -365,8 +359,11 @@ pub enum AttachmentType {
     /// [`symbolic_unreal::Unreal4LogEntry`].
     ///
     /// [`symbolic_unreal::Unreal4LogEntry`]: https://docs.rs/symbolic/*/symbolic/unreal/struct.Unreal4LogEntry.html
-    #[serde(rename = "unreal.logs")]
     UnrealLogs,
+
+    /// Unknown attachment type, forwarded for compatibility.
+    /// Attachments with this type will be dropped if `accept_unknown_items` is set to false.
+    Unknown(String),
```

Comment:
just curious what the benefit to implement `FromStr` instead of just using [`#serde(other)`](https://serde.rs/variant-attrs.html#other) here?

Reply:
`#serde(other)` does not work on non-unit enum variants, unfortunately. See https://github.com/serde-rs/serde/issues/1973.